### DISCUSSION TITLE
Bad speed and results? Is this normal

### DISCUSSION THREAD

**Discussion by DaiMaouYTS** [azuritecoin/OnnxDiffusersUI#74] [Q&A]
![000003-00 1boy, magic, fantasy, anime style, glowing sword, magic, high detail, bloom](https://user-images.githubusercontent.com/120827179/210050076-5594affb-015d-47f8-8043-8ab04b8e3024.png)

this one took around 3m and all of them take around the same time and if I change the settings they take even longer, I'm running this on RX 5700 XT, the usage seems to be at around 90% to 100% so I don't think that's the issue
the tags used were pretty simple `1boy, magic, fantasy, anime style, glowing sword, magic, high detail, bloom`
am I getting the right results and speed and is it supposed to be like that or is there something wrong with it? I used the UI version btw

**Comment by JJGall** [comment]
Yeah this is just the nature of Windows + AMD support right now. If you want speed you will have to switch to Linux or use an online service. One big reason for this is ROCm, AMD's compute stack, doesn't work on Windows. I have a 6800 and I ran this same prompt at 50 steps and it takes around 1 to 1 and a half minutes for a single image at 512x512. I usually get around 1.30s/it. If I use SD on Linux my card can get to 5 iterations a second which is a big difference.

**Comment by someguy1332** [comment]
I'd suggest tags that put emphasis on the face. Adding in beautiful face, detailed face and eyes, or even close up, can help improve the quality of the faces.  Other than that, you could do a lot worse, depending on what you need it for. More specialized models might help as well. 

As for the performance, yeah it sucks for me too. Maybe that will change in the future.

**Comment by ClashSAN** [comment]
Your steps, make them lower (8-15) and use dpms scheduler.
